src/api/balances.ts

```diff
@@ -56,15 +56,16 @@ export const getTokenBalance =
 export const useTokenBalance = (
   id: Maybe<string | u32>,
   address: Maybe<AccountId32 | string>,
+  options: { refetchInterval?: number } = {},
```

Comment:
Shouldn't options be an optional argument?

Reply:
removed the options because I dont need anymore